### PR TITLE
Update prizes and add favicon

### PR DESCRIPTION
**Pull request by gfting**

### CODE REVIEW

**Review by zannaw — COMMENTED** [review]
looks great!!

File: src/client/pages/hacker-guide.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ This guide is a list of everything you need to know to have the best possible vi
 
 ## Dear Hacker,
 
-We’re super excited to have you at our VandyHacks Summer Hackathon from [October 2nd-4th](https://calendar.vandyhacks.org)! In this document, we’ve included some basics to hackathons, logistics, event descriptions, mentor information, workshops, and prizes. Should you have any questions, feel free to email us at [info@vandyhacks.org](mailto:info@vandyhacks.org) or message us on [Facebook](https://www.facebook.com/vandyhacks).
+We’re super excited to have you at our VandyHacks Summer Hackathon from October 2nd-4th! In this document, we’ve included some basics to hackathons, logistics, event descriptions, mentor information, workshops, and prizes. Should you have any questions, feel free to email us at [info@vandyhacks.org](mailto:info@vandyhacks.org) or message us on [Facebook](https://www.facebook.com/vandyhacks).
```
Comment (aadibajpai) [inline src/client/pages/hacker-guide.md:5]: summer

File: src/client/pages/hacker-guide.md [hunk 2]
```diff
@@ -126,9 +140,10 @@ By attending VandyHacks, you agree to abide by the MLH Code of Conduct. If you
 
 # <a name="resources"></a>Resources
 
+-   [VandyHacks Workshops](https://learn.vandyhacks.org/)
 -   [Github Student Developer Pack](https://education.github.com/pack)
```
Comment (aadibajpai) [inline src/client/pages/hacker-guide.md:144]: GitHub

**Comment by aadibajpai** [comment]
thanks! I'll try to complete my pr before class today

**Pull request merged**